backend/src/test/java/com/morak/back/core/application/NotificationServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package com.morak.back.core.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.any;
+import static org.mockito.BDDMockito.anyLong;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.times;
+import static org.mockito.BDDMockito.verify;
+import static org.mockito.Mockito.doNothing;
+
+import com.morak.back.auth.domain.Member;
+import com.morak.back.core.domain.slack.SlackClient;
+import com.morak.back.core.domain.slack.SlackWebhook;
+import com.morak.back.core.domain.slack.SlackWebhookRepository;
+import com.morak.back.core.ui.dto.SlackWebhookCreateRequest;
+import com.morak.back.poll.domain.Poll;
+import com.morak.back.poll.domain.PollRepository;
+import com.morak.back.poll.domain.PollStatus;
+import com.morak.back.team.domain.Team;
+import com.morak.back.team.domain.TeamRepository;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class NotificationServiceTest {
+
+    @Mock
+    private SlackClient slackClient;
+
+    @Mock
+    private PollRepository pollRepository;
+
+    @Mock
+    private SlackWebhookRepository slackWebhookRepository;
+
+    @Mock
+    private TeamRepository teamRepository;
+
+    @InjectMocks
+    private NotificationService notificationService;
+
+    @Test
+    void 웹훅_URL을_등록한다() {
+        // given
+        Team team = Team.builder()
+            .id(1L)
+            .build();
+        String url = "https://hello.world";
+
+        given(teamRepository.findByCode(anyString())).willReturn(Optional.of(team));
+        doNothing().when(slackWebhookRepository).deleteByTeamId(anyLong());
```

Comment:
```suggestion
        given(teamRepository.findByCode(anyString())).willReturn(Optional.of(team));
        willDoNothing().given(slackWebhookRepository).deleteByTeamId(anyLong());
```
다른 메서드와 통일감있게 BDDMockito의 willDoNothing()을 사용하는게 어떨까요? (doNothing()은 Mockito꺼라서용👀)

Reply:
존재를 몰랐네요 ~ 알려주셔서 감사합니다 👍